src/main/resources/reference.conf

```diff
@@ -96,3 +97,8 @@ autoFreeze {
   autoFreezeAfter = 30 minutes
   autoFreezeCheckScheduler = 1 minute
 }
+
+jupyterContentSecurityPolicy {
```

Comment:
I wonder if we should name the config object `jupyterConfig`. Then we could access it like `"jupyterConfig.contentSecurityPolicy"`. I could envision us adding other jupyter configs here in the future.

Reply:
Agreed, that makes more sense